cmd/woodpecker-autoscaler/main.go

```diff
@@ -62,17 +62,23 @@ func run(ctx *cli.Context) error {
 
 	autoscaler := engine.NewAutoscaler(provider, client, config)
 
-	interval, err := time.ParseDuration(ctx.String("interval"))
+	config.MinAge, err = time.ParseDuration(ctx.String("min-age"))
+	if err != nil {
+		log.Error().Err(err).Msgf("cant parse agent min age, use default: %v", optionMinAgeDefault)
+		config.MinAge, _ = time.ParseDuration(optionMinAgeDefault)
+	}
+
+	config.Interval, err = time.ParseDuration(ctx.String("interval"))
 	if err != nil {
 		log.Error().Err(err).Msgf("cant parse reconciliation interval, use default: %v", optionIntervalDefault)
-		interval, _ = time.ParseDuration(optionIntervalDefault)
+		config.Interval, _ = time.ParseDuration(optionIntervalDefault)
 	}
 
 	for {
 		select {
 		case <-ctx.Done():
 			return nil
-		case <-time.After(interval):
+		case <-time.After(config.Interval):
```

Comment:
No need to add it to the config object IMO as its only used here. Renaming it to sth like `reconciliationInterval` would probably be more descriptive.